security_hub.tf

```diff
@@ -16,6 +16,36 @@ resource "aws_securityhub_organization_configuration" "default" {
   depends_on  = [aws_securityhub_organization_admin_account.default]
 }
 
+// AWS Security Hub - Management account enrollment
+resource "aws_securityhub_account" "management" {
```

Comment:
This should be moved up under the section: `// AWS Security Hub - Management account configuration`